docs/design/proposals/status-conditions.md

```diff
@@ -198,18 +198,84 @@ Things to note:
 
 ##### Resource Graph Instance
 
-This one is currently pretty tricky because it accepts conditions to be defined
-inside the RGD Instance. But at minimum and by default the controller adds:
+The Resource Graph Instance currently uses a single condition:
 
 - InstanceSynced
 
-To be able to use a condition manager, we would need a signal from the provided
-status shape to understand if the defined condition is a dependent condition or
-just informational.
+Based on analysis of the instance reconciler, the major sections of work can be
+broken down into the following sub-conditions that align with the reconciliation
+phases:
+
+```
+Ready
```

Comment:
i've been thinking about adding a feature that let's kro validate it's permissions to manage RGD instances (using [SelfSubjectRuleReview](https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/self-subject-rules-review-v1/)), maybe we need a condition "Authorized" (there is a probably a better name?) which could surface state describing client impersonation and subject rule reviews evaluation results?

Reply:
@a-hilaly I think this deserves an own issue, right? do you already have something for that? if not I can create one and we can discuss further there

Reply:
I took a jab at this in https://github.com/kro-run/kro/pull/646. Feel free to take a look anytime